src/Common/Query/GeocodeQuery.php

```diff
@@ -19,14 +19,14 @@
 /**
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
-final class GeocodeQuery implements Query
```

Comment:
This hurts a little but Im fine with it =)

Reply:
Thank you for that. I am glad that you will at least allow this.